cmd/jaeger/config-cassandra.yaml

```diff
@@ -34,6 +34,7 @@ extensions:
         cassandra:
           schema: 
             keyspace: "jaeger_v1_dc1"
+            create: "${env:SKIP_APPLY_SCHEMA:-false}"
```

Comment:
```suggestion
            create: "${env:CASSANDRA_CREATE_SCHEMA:-false}"
```

Reply:
`SKIP_APPLY_SCHEMA` means that we are skipping schema creation before test, but this context is missing in the name. I think `CASSANDRA_CREATE_SCHEMA` would also be similar to this because we haven't provided context in it.
Should we use `AUTO_CREATE_SCHEMA` to provide the context that it is automatically created, without any script required?

Reply:
got it, just wanted to understand the reason. Changed it. Thanks